Automattic/wp-calypso
# Update variant dropdown for multiple-year plans to display discounts

Related to p1685720048306269-slack-C02LK1W8T4Z

## Proposed Changes

* Add back the discount percentage for Jetpack plans to the variant dropdown (for 2-year and 3-year plans). The idea behind this is to let the users know that the 2-year plans have additional discounts compared to the regular discount from an Intro Offer

## Testing Instructions



* Visit the Calypso live link
* Go to `/checkout/jetpack/jetpack_security_t1_yearly#step2`
* Confirm that the two-year plan displays the discount

Before | After
--- | ---
<img width="484" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/37efe784-655a-4470-90c3-183138e25aa9"> | <img width="487" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/6d3e40ab-4df4-422b-bdd9-36e8ed516bc4">

## Pre-merge Checklist
